pkgs/desktops/plasma-5/plasma-desktop/default.nix

```diff
@@ -115,10 +118,20 @@ mkDerivation {
   patches = [
     ./hwclock-path.patch
     ./tzdir.patch
+    ./kcm-access.patch
   ];
-  CXXFLAGS = [
-    ''-DNIXPKGS_HWCLOCK=\"${lib.getBin util-linux}/sbin/hwclock\"''
-  ];
+  CXXFLAGS =
+    let
+      # run gsettings with desktop schemas for using in kcm_accces kcm
+      gsettings-wrapper = writeShellScript "gsettings-wrapper" ''
+        export XDG_DATA_DIRS=$XDG_DATA_DIRS:${gsettings-desktop-schemas.out}/share/gsettings-schemas/${gsettings-desktop-schemas.name}
+        exec ${glib}/bin/gsettings "$@"
+      '';
+    in
+    [
+      ''-DNIXPKGS_HWCLOCK=\"${lib.getBin util-linux}/sbin/hwclock\"''
```

Comment:
This is under sbin, but does it really require the suid bit? If so this needs to be the wrapper path because the store file does not have that.

Reply:
it was there before i chaged i don't know
i just formatted with my patch

Reply:
sbin is a symlink to bin so we can just change that to avoid further questions